api_tests/tests/rfc003_schema.ts

```diff
@@ -1,80 +1,24 @@
 import { Actor } from "../src/actors/actor";
-import { expect } from "chai";
-import "chai/register-should";
-import "../src/setup_chai";
 import * as sirenJsonSchema from "../siren.schema.json";
 import * as swapPropertiesJsonSchema from "../swap.schema.json";
 import { twoActorTest } from "../src/actor_test";
 import { createDefaultSwapRequest, DEFAULT_ALPHA } from "../src/utils";
-import {
-    Action,
-    EmbeddedRepresentationSubEntity,
-    Entity,
-    Link,
-} from "comit-sdk";
+import { Action, EmbeddedRepresentationSubEntity, Link } from "comit-sdk";
+import { extendSchemaMatcher } from "../src/schema_matcher";
+
+extendSchemaMatcher();
```

Comment:
Is there a more global place where we could do this?

Reply:
Let's try :)